upload/views/helpers.py

```diff
@@ -1,8 +1,13 @@
+import logging
```

Comment:
This file can now be deleted right?

Reply:
I can delete `get_repository_and_owner_from_slug_and_commit`, my bad I thought I had removed it! ty for catching.